rust/ql/test/utils/InlineFlowTest.qll

```diff
@@ -10,15 +10,16 @@ private import codeql.rust.dataflow.internal.DataFlowImpl
 private import codeql.rust.dataflow.internal.TaintTrackingImpl
 private import internal.InlineExpectationsTestImpl as InlineExpectationsTestImpl
 
+// Holds if the target expression of `call` is a path and the string representation of the path is `name`.
+private predicate callTargetName(CallExpr call, string name) {
+  call.getExpr().(PathExpr).getPath().toString() = name
```

Comment:
I think we should define the `toString` of `PathExpr` to be `result = this.getPath().toString()` .

Reply:
That sounds good to me. I'll update the PR to do that instead :)

Reply:
In general, we ought to implement a lot of more sensible `toString`s, but that is of course not for this PR...

Reply:
I've added the suggested `toString` on `PathExpr`. Note that this added two inconsistencies [here](https://github.com/github/codeql/pull/17995/files#diff-c723bd134cec4d34136806d1f7d90c9df8907e186e5de8a0d7ce121b92812cb3) because the `Path` nodes on the last two lines in [gen_path_expr.rs](https://github.com/github/codeql/blob/main/rust/ql/test/extractor-tests/generated/PathExpr/gen_path_expr.rs) doesn't have a string representation. I don't know if this is a bug in the extractor? When I explore the AST for that file I also see that the last two `let` statements doesn't have an initializer which seems odd?

Reply:
If it's too much hassle to address this in the current PR, then we can also accept the inconsistencies and fix them in a follow-up.

Reply:
Thanks, that makes sense. I've accepted the inconsistencies for now.